framework/test/mocha/functional/http/get/transactions.js

```diff
@@ -357,6 +357,129 @@ describe('GET /api/transactions', () => {
 						});
 					});
 			});
+
+			describe('asset field', () => {
+				it('using type 0 should return asset field with correct properties', async () => {
+					return transactionsEndpoint
+						.makeRequest({ type: transactionTypes.SEND }, 200)
+						.then(res => {
+							expect(res.body.data).to.not.empty;
+
+							res.body.data.map(transaction => {
+								expect(transaction).to.have.property('asset');
```

Comment:
This check is redundant because `Object.keys(transaction.asset)` would throw exception if asset is undefined.